DigitalLearningSolutions.Web/Helpers/CustomClaimHelper.cs

```diff
@@ -72,5 +72,11 @@ public static bool IsDelegateOnlyAccount(this ClaimsPrincipal user)
             return user.GetAdminId() == null
                    && (user.GetCustomClaimAsBool(CustomClaimTypes.LearnUserAuthenticated) ?? false);
         }
+
+        public static bool HasAdminCentrePermissions(this ClaimsPrincipal user)

```

Comment:
Super minor, but this type of user is called "centre admin" rather than "admin centre"